common.py

```diff
@@ -1,199 +1,120 @@
 import jinja2
-from dataclasses import dataclass, fields, field, is_dataclass
-from enum import Enum
-from typing import Optional, Any, Type, TypeVar, cast
-from typing import Mapping
-
-TFT_TOOLS_IMG = "quay.io/wizhao/tft-tools:latest"
-TFT_TESTS = "tft-tests"
-
-MEASURE_POWER_PLUGIN = "measure_power"
-MEASURE_CPU_PLUGIN = "measure_cpu"
-VALIDATE_OFFLOAD_PLUGIN = "validate_offload"
-
 
-@dataclass
-class Result:
-    out: str
-    err: str
-    returncode: int
+from dataclasses import fields
+from dataclasses import is_dataclass
+from enum import Enum
+from typing import Any
+from typing import Optional
+from typing import Type
+from typing import TypeVar
+from typing import cast
```

Comment:
OOC, why specify these one per line rather than in a single import? Is this enforced by some linter?

Reply:
I try to give the reasoning in the commit message. If that is unclear, then at least the commit message should be improved. But let me explain here:

it's not enforced by a linter. There is https://pypi.org/project/flake8-import-order/ , however that does not enforce single-line imports. `flake8-import-order` is also not followed, because it has additional style suggestions, that I found cumbersome.

The only reason why I did this, is because IMO this is more readable and nicer with future changes (a diff adds/removes lines for the import that changes, nothing else). I guess, that opinion is not unversally shared, if you see https://stackoverflow.com/questions/15011367/python-module-import-single-line-vs-multi-line .

In short, just my preference. But we can adopt another style, if that is preferred.

Reply:
I see, sorry to make you repeat yourself, I missed the commit message. Makes sense to me, I like the multi-line